### PR TITLE
feat: add rotated linknet_resnet18 tensorflow ckpts

### PR DESCRIPTION
**Pull request by charlesmindee**
Following #816, this PR adds the checkpoints of the trained linket_resnet18 on rotated samples.
Any feedback is welcome!

### CODE REVIEW

**Comment by fg-mindee** [comment]
There are conflicts to be resolved if we are to move forward with the PR :)

**Comment by fg-mindee** [comment]
ping @charlesmindee :)

**Review by fg-mindee — CHANGES_REQUESTED** [review]
With the recent changes on the loss, I think there has some conflicts to resolve before moving on with this!

**Review by fg-mindee — CHANGES_REQUESTED** [review]
isort isn't happy but the rest is good :+1:

**Review by fg-mindee — APPROVED** [review]
Thanks!
